src/main/resources/META-INF/accesstransformer.cfg

```diff
@@ -75,7 +75,9 @@ public net.minecraft.client.particle.ParticleEngine m_107378_(Lnet/minecraft/cor
 public net.minecraft.client.particle.ParticleEngine m_107381_(Lnet/minecraft/core/particles/ParticleType;Lnet/minecraft/client/particle/ParticleProvider;)V # register
 public net.minecraft.client.particle.ParticleEngine$SpriteParticleRegistration
 public net.minecraft.client.renderer.GameRenderer m_109128_(Lnet/minecraft/resources/ResourceLocation;)V # loadEffect
+public net.minecraft.client.renderer.LevelRenderer m_172993_(Lnet/minecraft/client/renderer/RenderType;Lcom/mojang/blaze3d/vertex/PoseStack;DDDLcom/mojang/math/Matrix4f;)V
```

Comment:
This needs to have the mapped name of the method (`renderChunkLayer`) in a comment after the declaration.

As a thought, is it possible and viable to avoid this AT (and opening up `renderChunkLayer` to everyone) by passing in a method reference to it to the `LevelRendererAdapter` hooks?